sandbox/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-﻿cmake_minimum_required (VERSION 3.13)
+cmake_minimum_required (VERSION 3.13)
```

Comment:
I was a bit puzzled to see this change. For all intents and purposes, these lines look identical :zany_face:.

The magic is that the old version contains UTF-8 BOM, whereas the new one doesn't. Not that we needed the BOM; but CMake works fine either way, so this change is unnecessary. IMHO, the fewer files in the PR, the better.

Reply:
Yes sure! But honestly, how can I add it back? :)

Reply:
Hmm... that's a good question :joy:. Does BOM get removed on checkout on your system? If not, you can probably check out an old commit, copy the file (or rename it) to retain the binary copy, and then try to commit it again.

Worst case I can add it back from an old commit and then you'll cherry-pick if from my fork.

Reply:
Copy-pasting from the main branch worked perfectly, thanks!